libeos-payg/service.c

```diff
@@ -172,8 +241,8 @@ epg_service_startup_async (GssService          *service,
       return;
     }
 
-  epg_manager_new (enabled, NULL, NULL,
-                   cancellable,
+  GCancellable *cancellable = g_task_get_cancellable (task);
```

Comment:
Maybe worth adding a comment here like `/* No provider plugin could be initialised. Fall back to #EpgManager. */`?

Reply:
Or maybe a bit further up by the `g_debug()` that kicks this all off.